lib/Exporter/ONNXModelWriter.cpp

```diff
@@ -650,6 +650,15 @@ llvm::Error ONNXModelWriter::writeReshape(const ReshapeNode *node,
   return writeAll("Reshape", node, graph);
 }
 
+llvm::Error ONNXModelWriter::writeBucketize(const BucketizeNode *node,
```

Comment:
this is actually interesting. Bucketize is C2 op, but we serialize that to ONNX.
@putivsky is this the right way to go?

Reply:
yeah I found that a bit off: had to add this method to workaround a linkage error (see above), but I am not sure this is the way to go

Reply:
We will be able to load not "kosher" operators from ONNX format back to Glow, I'm working on it.